megalinter/descriptors/dart.megalinter-descriptor.yml

```diff
@@ -68,7 +68,8 @@ linters:
       ## stable dart sdk: https://dart.dev/get-dart#release-channels
       ## install alpine-pkg-glibc (glibc compatibility layer package for Alpine Linux)
       dockerfile:
-        - ARG DART_VERSION='2.8.4'
+        # renovate: datasource=dart-version depName=dart
+        - ARG DART_VERSION='3.6.1'
```

Comment:
Seems only like a comment in the descriptor, not copied over to the Dockerfiles

Reply:
Done!